tests/e2e/utils/payments.js

```diff
@@ -253,11 +254,21 @@ export async function confirmCardAuthentication(
  */
 export async function setupProductCheckout(
 	billingDetails,
-	lineItems = [ [ config.get( 'products.simple.name' ), 1 ] ]
+	lineItems = [ [ config.get( 'products.simple.name' ), 1 ] ],
+	currency = null
 ) {
 	const cartItemsCounter = '.cart-contents .count';
 
-	await shopper.goToShop();
+	if ( currency ) {
+		await page.goto(
```

Comment:
Nit: Should we add this to the [utils/flows.js](https://github.com/Automattic/woocommerce-payments/blob/16385759ab5533e5970c3f01f0b9f540256e1fb1/tests/e2e/utils/flows.js)? It seems like there's similar functionality there.

Nit: We can also update this to the following:
```javascript
...
const { shopper, uiUnblocked, SHOP_PAGE } = require( '@woocommerce/e2e-utils' );
...
await page.goto( SHOP_PAGE + `/?currency=${ currency }`, {
  waitUntil: 'networkidle0',
} );
...
```

Reply:
Thanks for the feedback. I've restored the original `setupProductCheckout` function and introduced a new flow `goToShopWithCurrency`: https://github.com/Automattic/woocommerce-payments/pull/7804/commits/d3d5e23d1b2b03387acfc482af6e5af1bc019750.